src/modules/order/index.ts

```diff
@@ -53,29 +53,28 @@ export async function getOrderDetails (orderId: string): Promise<Order | Respons
 
       order.parts = orderShipGroup as OrderPart[]
 
-      response = order
+      response = Promise.resolve(order)
     } else {
-      response = {
+      response = Promise.reject({
```

Comment:
I think we should return it as a common pattern

Reply:
Improved the pattern to return from the oms-api package as soon as the promise gets rejected.